src/main/sarl/fr/utbm/info/vi51/worldswar/environment/envobject/AntHill.java

```diff
@@ -46,6 +54,30 @@ public void setColony(Colony colony) {
 		this.colony = colony;
 	}
 
+	/**
+	 * @return the spawnCooldown
+	 */
+	public int getSpawnCooldown() {
+		return this.spawnCooldown;
+	}
+
+	/**
+	 * @param spawnCooldown
+	 *            the spawnCooldown to set
+	 */
+	public void setSpawnCooldown(int spawnCooldown) {
```

Comment:
Je verrai plutôt une méthode du genre :
public void resetSpawnCooldown(){
this.spawnCooldown = SPAWN_COOLDOWN ;
}
pour être sur qu'on ne puisse pas setter n'importe quel cooldown